packages/@o3r/schematics/src/modules/modules.constants.ts

```diff
@@ -0,0 +1,35 @@
+import type { PackageJson } from 'type-fest';
+
+/** Package as return by the NPM Registry */
+export type NpmRegistryPackage = Pick<PackageJson, 'name' | 'description' | 'version' | 'keywords'> &
```

Comment:
Why do you pack `name` to override it ?

Reply:
To enforce the support of the type of packageJson's name type (actually `string | undefined` :D).
It is not exactly overridden, it is extended (to remove the `undefined` part)